webview-ui/src/components/ui/comfirm-dialog.tsx

```diff
@@ -0,0 +1,58 @@
+import { Dialog, DialogContent, DialogTitle } from "./dialog"
+import { VSCodeButton } from "@vscode/webview-ui-toolkit/react"
+import { useCallback } from "react"
+
+export interface ConfirmDialogProps {
+	show: boolean
+	icon: string
+	title?: string
+	message: string
+	onResult: (confirm: boolean) => void
+	onClose: () => void
+}
+export const ConfirmDialog = ({ onResult, onClose, icon, show, title, message }: ConfirmDialogProps) => {
```

Comment:
Not a blocker, but we might want to use the official shadcn version of this component:

```bash
npx shadcn@latest add alert-dialog
```

Reply:
Many `radix-ui` components are already in use, it also provides an `Alert Dialog`, and we only use the `Dropdown` from the `vscrui` , I’m considering switching these to `radix-ui`.

Reply:
@System233  i think we can use shadcn too, shadcn built on radix-ui with tailwind theme support, and @cte  already config theme for tailwind and shadcn component

Reply:
👌